sdk/eventhub/azure-eventhubs/examples/send_event_data_batch.py

```diff
@@ -40,25 +35,13 @@ def create_batch_data(producer):
     return event_data_batch
 
 
-try:
-    if not HOSTNAME:
-        raise ValueError("No EventHubs URL supplied.")
-
-    client = EventHubClient(host=HOSTNAME, event_hub_path=EVENT_HUB, credential=EventHubSharedKeyCredential(USER, KEY),
-                            network_tracing=False)
-    producer = client.create_producer()
-
-    try:
-        start_time = time.time()
-        with producer:
-            event_data_batch = create_batch_data(producer)
-            producer.send(event_data_batch)
-    except:
-        raise
-    finally:
-        end_time = time.time()
-        run_time = end_time - start_time
-        logger.info("Runtime: {} seconds".format(run_time))
-
-except KeyboardInterrupt:
-    pass
+client = EventHubClient(host=HOSTNAME, event_hub_path=EVENT_HUB, credential=EventHubSharedKeyCredential(USER, KEY),
+                        network_tracing=False)
+producer = client.create_producer()
+start_time = time.time()
+with producer:
+    event_data_batch = create_batch_data(producer)
+    producer.send(event_data_batch)
+end_time = time.time()
+run_time = end_time - start_time
+print("Runtime: {} seconds".format(run_time))
```

Comment:
suggest collapsing these three lines to `print("Runtime: {} seconds".format(time.time() - start_time)` (and elsewhere) Examples should reserve the most lines for what's actually being demonstrated and try to reduce tangential boilerplate.

Reply:
that's a very good point, never thought this would have a impact.
code is updated. I'll pay more attention to the examples.